pymatbridge/publish.py

```diff
@@ -1,5 +1,5 @@
-import IPython.nbformat.v4 as nbformat
-from IPython.nbformat import write as nbwrite
+import nbformat.v4 as nbformat
```

Comment:
lol, yeah, figured that out about 30 seconds after that pull request. I was a bit awkward/slow canceling the pull request and redoing it. 


Joe

> On Aug 29, 2015, at 9:28 AM, Steven Silvester <notifications@github.com> wrote:
> 
> In pymatbridge/publish.py <https://github.com/arokem/python-matlab-bridge/pull/212#discussion_r38261998>:
> 
> > @@ -1,5 +1,5 @@
> > -import IPython.nbformat.v4 as nbformat
> > -from IPython.nbformat import write as nbwrite
> > +import nbformat.v4 as nbformat
> This should be a try/except ImportError with the new format first to accommodate both versions.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/arokem/python-matlab-bridge/pull/212/files#r38261998>.
>